python/ml4ir/applications/ranking/tests/data/configs/linear_model/model_config.yaml

```diff
@@ -0,0 +1,39 @@
+architecture_key: dnn
+layers:
+  - type: dropout
+    name: first_dropout
+    rate: 0.0
+  - type: dense
+    name: final_dense
+    units: 1
+    activation: null
+optimizer:
+  key: adam
+  gradient_clip_value: 5.0
+lr_schedule:
+  key: constant
+  learning_rate: 0.01
+
+
+#Below is an example of defining other learning rate schedules.
```

Comment:
Better remove all these comments ;-)

Reply:
Will do. I had left it as a reference to use other `lr_schedule` in case people were interested.